connection-schema.md

```diff
@@ -0,0 +1,72 @@
+# Connection Parameter Schema in Generator Object
+```json
+{
+  "connection": {
+    "simple-auth": {
+      "credential": {
+        "client": {
+          "id": "CLIENT_ID",
+          "secret": "CLIENT_SECRET",
+          "redirectUri": "REDIRECT_URI"
+        },
+        "auth": {
+          "tokenHost": "HOST_URL",
+          "tokenPath": "HOST_PATH"
+        },
+        "options": {
+          "bodyFormat": "json"
+        }
+      },
+      "authorizarionUri": {
+        "redirect_uri": "REDIRECT_URI",
+        "scope": "notifications",
+        "state": "3(#0/!~"
+      },
+      "tokenConfig": {
+        "username": "USERNAME",
+        "passeword": "PASSWORD"
+      },
+      "tokenObject": {
+        "access_token": "<Token>",
+        "refresh_token": "<Token>",
+        "expires_in": "t3600"
+      },
+      "method": "authorizationCode || ownerPassword || clientCredentials",
+      "tokenFormat": "Bearer {accessToken}",
+      "salesforce": true
+    }
+  }
+}
+```
+## Examples
+### Argus
+```json
+{
+  "connection": {
+    "simple_oauth": {
+      "credentials": {
+        "client": {
+        },
+        "auth": {
+          "tokenHost": "ARGUS_HOST",
+          "tokenPath": "ARGUS_PATH"
```

Comment:
can you give an example of this argus path (but don't use our *actual* argus host name)